### PR TITLE
add release notes for 0.2.10

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR adds release notes for 0.2.10

### CODE REVIEW

File: docs/release/release_0_2_10.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Announcement: napari 0.2.10
+===========================
+
+We're happy to announce the release of napari 0.2.10!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
```
Comment (jni) [inline docs/release/release_0_2_10.rst:9]:
```suggestion

This is a bugfix release to address issues that snuck through in 0.2.9.
```

File: docs/release/release_0_2_10.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Bugfixes
+*******
```
Comment (jni) [inline docs/release/release_0_2_10.rst]:
```suggestion
********
```

**Review by jni — APPROVED** [review]

**Comment by jni** [comment]
Approved, with some optional minor comments!

File: docs/release/release_0_2_10.rst (hunk 1 above)
Comment (sofroniewn) [reply docs/release/release_0_2_10.rst:9]: made change

File: docs/release/release_0_2_10.rst (hunk 2 above)
Comment (sofroniewn) [reply docs/release/release_0_2_10.rst]: changes

**Comment by sofroniewn** [comment]
Thanks @jni - changes made. I will merge once tests all pass, and then make `0.2.10rc0`

**Pull request merged**